.github/workflows/ci.yaml

```diff
@@ -25,6 +25,6 @@ jobs:
         with:
```

Comment:
fwiw, i think actions/setup-go is not needed because the bazel action will get yer toolchain

Reply:
thank you!